client/components/layout/Footer.tsx

```diff
@@ -1,10 +1,216 @@
+  };
+  @media screen and (max-width: ${sizes.small}px) {
+    display: flex;
+  }
+`
+
+const Copyright = styled.p`
+  padding: 10px 0px;
+  margin: 5px;
+  font-size: 12px;
+
+  @media screen and (max-width: ${sizes.large}px) {
+    display: none;
+  }
+`
+
+const ContactItem = styled.div`
+  display: flex;
+`
+
+const Circle = styled.div`
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  background-color: ${props => props.theme.colors.mainLightFontColor};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem 0rem;
+`
+
+const Icon = styled.i`
+  color: ${props => props.theme.colors.main};
+`
+
+const CircleInfo = styled.p`
+  padding: 0px 10px;
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+
+  @media screen and (max-width: ${sizes.large}px) {
+    font-size: 13px;
+    padding: 0px 5px;
+  };
+  @media screen and (max-width: ${sizes.small}px) {
+    display: none;
+  }
+`
+
+const SquareButtonContainer = styled.div`
+  display: flex;
+
+  @media screen and (max-width: ${sizes.small}px) {
+    display: flex;
+    flex-direction: column;
+  }
+`
+
+const SquareButton = styled.button`
+  border-radius: 3px;
+  border: none;
+  margin: 5px;
+  padding: 5px;
+  width: 3rem;
+  height: 3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const AboutItem = styled.p`
+  width: 35rem;
+
+  @media screen and (max-width: ${sizes.large}px) {
+    width: auto;
+`
+
+const AboutHead = styled.p`
+  font-size: 18px;
+  font-family: ${props => props.theme.fonts.menu};
+
+  @media screen and (max-width: ${sizes.large}px) {
+    display: flex;
+    font-size: 14px;
+  };
+  @media screen and (max-width: ${sizes.small}px) {
+    display: none;
+  }
+`
+
+const AboutText = styled.p`
+  font-size: 13px;
+  padding: 10px 0px;
+
+  @media screen and (max-width: ${sizes.large}px) {
+    display: flex;
+    font-size: 11px;
+  };
+  @media screen and (max-width: ${sizes.small}px) {
+    display: none;
+  }
+`
+
 
 const Footer = () => {
   return (
-    <footer>
-      Footer
-    </footer>
+    <FooterContainer>
+      <FooterMaxWidthContainer>
+        <Column>
+          <Logo src='/static/logo-main-white.png' alt='Logo'/>
+          <Menu>
+            <MenuItem href=''target='_blank'>Menu</MenuItem>
+            <MenuItem href='' target='_blank'>Zakázky</MenuItem>
+            <MenuItem href='' target='_blank'>Náš příběh</MenuItem>
+            <MenuItem href='' target='_blank'>Otázky</MenuItem>
+            <MenuItem href='' target='_blank'>Kontakty</MenuItem>
+          </Menu>
+          <Copyright>
+            Copyright ©2020 Zlatnictví Vaněk<br/>Zásady ochrany osobních údajů
+          </Copyright>
+        </Column>
+        <Column>
+          <ContactItem>
+            <Circle><Icon className='fas fa-map-marker-alt fa-lg'/></Circle>
+            <CircleInfo>Pražská 164<br/>280 02 Kolín</CircleInfo>
+            </ContactItem> 
+          <ContactItem>
```

Comment:
I think there's a way to make this a link that calls this number or at least enters it in the phone